src/clj/game/cards/programs.clj

```diff
@@ -2785,6 +2785,55 @@
                                                  (trigger-event state :corp :no-action))}])
                                 (make-run eid :archives (get-card state card)))}]})
 
+(defcard "Sneakdoor Prime A"
+         {:abilities [{:cost [:click 2]
+                       :prompt "Choose a server"
+                       :choices (req (cancellable
+                                       (->> runnable-servers
+                                            (map unknown->kw)
+                                            (filter is-remote?)
+                                            (map remote->name))))
+                       :msg "make a run on a remote server"
+                       :makes-run true
+                       :async true
+                       :effect (effect (register-events
+                                         card
+                                         [{:event :pre-successful-run
+                                           :duration :end-of-run
+                                           :unregister-once-resolved true
+                                           :prompt "Choose a server"
+                                           :choices (req (->> servers
+                                                              (map unknown->kw)
+                                                              (filter is-central?)
+                                                              (map central->name)))
+                                           :msg (msg "change the attacked server to " target)
+                                           :effect (req (swap! state assoc-in [:run :server] [(unknown->kw target)])
+                                                        (trigger-event state :corp :no-action))}])
```

Comment:
Is this necessary?

Reply:
maybe not - I copied it from Sneakdoor Beta

Reply:
Removed as I don't think this was necessary